org.ekstep.reviewercomments-1.0/editor/plugin.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Plugin to get comments from portal
+ * @class reviewerComments
+ * @constructor
+ * @extends org.ekstep.contenteditor.basePlugin
+ * @author Revathi P 
+ * @listens stage:select
+ */
+
+org.ekstep.contenteditor.basePlugin.extend({
+    isApiCalled: false,
+    initialize: function () {
+        ecEditor.addEventListener("stage:select", this.initShowComments, this);
+        this.initShowComments();
+    },
+    initShowComments: function (event) {
+        var instance = this;
+        this.callAPI(function () {
+            setTimeout(function () {
```

Comment:
SetTimeout should not use for this use-case. Remove this code.

Reply:
Removed